cdist/conf/type/__consul_template_template/manifest

```diff
@@ -66,7 +69,7 @@ for param in *; do
    esac
 done
 printf '}\n'
-) | \
+} | \
```

Comment:
@thriqon @asteven Using { } instead of ( ) does not work here either, because of pipe it is still executed in subshell. So I moved variable assignment before this. Please review.

Reply:
@asteven Can you verify that this works as expected?

Reply:
I will fix it when/if it breaks something. Move along.